pkg/dataServer/dataServer_test.go

```diff
@@ -25,18 +23,18 @@ func TestDataServer(t *testing.T) {
 	exitCh := make(chan int)
 	logSetup := util.SetupLogger()
 	logger := logSetup("debug")
-	ctx, cfg, cleanup := tcontext.CreateTestContext(t)
-	defer t.Cleanup(cleanup)
+	cfg := tcontext.CreateTestConfig(t)
```

Comment:
There is already an existing function so lets delete CreateTestConfig and that one instead here and in the other places.
```suggestion
	cfg := config.TestConfig(t)
```